gateway/api_definition.go

```diff
@@ -595,6 +599,25 @@ func (a APIDefinitionLoader) replaceSecrets(in []byte) []byte {
 		}
 	}
 
+	if strings.Contains(input, prefixConsul) {
```

Comment:
can we move this into a separate function, so that `goto` can be avoid and can do the control with return

Reply:
done